src/actions/__tests__/resetActions.js

```diff
@@ -3,8 +3,8 @@ import moxios from 'moxios';
 import thunk from 'redux-thunk';
```

Comment:
@ogiste is this Valerie's work?

Reply:
Yes, I just needed to make some  adjustments to her formatting to match the eslint requirements,

Reply:
okay